jupyter_sphinx/ast.py

```diff
@@ -0,0 +1,351 @@
+"""Manipulating the Sphinx AST with Jupyter objects"""
+
+import os
+import json
+
+import docutils
+from docutils.parsers.rst import Directive, directives
+from docutils.nodes import math_block
+from sphinx.util import parselinenos
+from sphinx.addnodes import download_reference
+
+import ipywidgets.embed
+import nbconvert
+
+import jupyter_sphinx as js
```

Comment:
It's a cosmetic pont, but I prefer to not introduce abbreviations like this, except for the very standard `np` or `plt`.

At the same time, seeing where this is used, it seems that

```suggestion
from jupyter_sphinx import logger
```

would work even better.

Reply:
ah - the reason I was doing this is because the logger gets initialized by Sphinx so it's not importable at first. I'll lazily import the logger as you show above but inside the run function

Reply:
Ah, thanks for the explanation.